src/testUtils.ts

```diff
@@ -455,7 +455,11 @@ function targetArgs(testconfig: TestConfig): Array<string> {
 			// in running all the test methods, but one of them should call testify's `suite.Run(...)`
 			// which will result in the correct thing to happen
 			if (testFunctions.length > 0) {
-				params = params.concat(['-run', util.format('^(%s)$', testFunctions.join('|'))]);
```

Comment:
I had to change this because enclosing the full subtest name causes the match to fail and the test to not execute. The [issue](https://github.com/microsoft/vscode-go/issues/2144) that introduced the brackets mentions that this is an issue for running multiple tests. With an explicit test count, this should maintain the current functionality.